test/test_jit.py

```diff
@@ -17192,6 +17196,12 @@ def fn(x=None):
         self.checkScript(fn, ((3, 4),))
         self.checkScript(fn, ())
 
+    def test_tensor_clone(self):
```

Comment:
can you add the test here to https://github.com/pytorch/pytorch/blob/master/test/test_jit.py#L8107 ? I think you just need to add some tensor constructors to `lists`. you may need to only add them when we're testing `tensor` and not `as_tensor`, and you may need to invoke `with freeze_rng_state():` if you're using rand initializers.

Reply:
ok this part should be done in the latest commit.